testsuite/tests/tool-ocamldoc-html/Makefile

```diff
@@ -33,10 +33,11 @@ default:
 	fi
 
 .PHONY: run
-run: *.mli
-	@for file in *.mli; do \
+run: *.mli *.ml
+	@for file in *.ml *.mli; do \
 	  printf " ... testing '$$file'"; \
 	  F="`basename $$file .mli`"; \
+	  F="`basename $$F .ml`"; \
```

Comment:
Are you overwriting the previous `F` variable here? I'm a bit confused, are the `.mli` still tested?